aces/Loris
# [candidate_parameters] Allow configuration of which status means comment required

## Brief summary of changes

- Add new column to participant_status_options allowing the configuration of which status means that the comment field is required.

#### Testing instructions (if applicable)
1. Run the patch
2. Open candidate parameters>participant status
3. Change the status and confirm that the ones with 1 in commentRequired force the comment to be required

[CCNA OVERRIDE PR](https://github.com/aces/CCNA/pull/6311)
